public/blog-posts-data.json

```diff
@@ -1 +1 @@
":"Evolving the Node.js Brand","author":"Emily Tanaka-Delgado","date":"2011-07-11T19:02:45.000Z","category":"uncategorized","slug":"/blog/uncategorized/evolving-the-node-js-brand"},{"title":"Node v0.5.0 (Unstable)","author":"ryandahl","date":"2011-07-06T09:23:17.000Z","category":"release","slug":"/blog/release/v0.5.0"},{"title":"Node v0.4.9","author":"ryandahl","date":"2011-06-29T18:41:05.000Z","category":"release","slug":"/blog/release/v0.4.9"},{"title":"Porting Node to Windows With Microsoft’s Help","author":"ryandahl","date":"2011-06-23T22:22:58.000Z","category":"uncategorized","slug":"/blog/uncategorized/porting-node-to-windows-with-microsofts-help"},{"title":"Node v0.4.8","author":"ryandahl","date":"2011-05-21T14:06:00.000Z","category":"release","slug":"/blog/release/v0.4.8"},{"title":"npm 1.0: Released","author":"Isaac Schlueter","date":"2011-05-01T15:09:45.000Z","category":"npm","slug":"/blog/npm/npm-1-0-released"},{"title":"Trademark","author":"ryandahl","date":"2011-04-29T08:54:18.000Z","category":"uncategorized","slug":"/blog/uncategorized/trademark"},{"title":"Node Office Hours Cut Short","author":"ryandahl","date":"2011-04-28T16:04:35.000Z","category":"uncategorized","slug":"/blog/uncategorized/node-office-hours-cut-short"},{"title":"Node v0.4.7","author":"ryandahl","date":"2011-04-23T07:47:55.000Z","category":"release","slug":"/blog/release/v0.4.7"},{"title":"Node v0.4.6","author":"ryandahl","date":"2011-04-14T12:00:30.000Z","category":"release","slug":"/blog/release/v0.4.6"},{"title":"npm 1.0: link","author":"Isaac Schlueter","date":"2011-04-07T00:40:33.000Z","category":"npm","slug":"/blog/npm/npm-1-0-link"},{"title":"Development Environment","author":"ryandahl","date":"2011-04-05T03:16:27.000Z","category":"uncategorized","slug":"/blog/uncategorized/development-environment"},{"title":"node v0.4.5","author":"ryandahl","date":"2011-04-02T09:04:58.000Z","category":"release","slug":"/blog/release/v0.4.5"},{"title":"Node v0.4.4","author":"ryandahl","date":"2011-03-26T15:58:45.000Z","category":"release","slug":"/blog/release/v0.4.4"},{"title":"jobs.nodejs.org","author":"ryandahl","date":"2011-03-25T06:05:22.000Z","category":"uncategorized","slug":"/blog/uncategorized/jobs-nodejs-org"},{"title":"npm 1.0: Global vs Local installation","author":"Isaac Schlueter","date":"2011-03-24T06:07:13.000Z","category":"npm","slug":"/blog/npm/npm-1-0-global-vs-local-installation"},{"title":"Office Hours","author":"ryandahl","date":"2011-03-24T04:42:47.000Z","category":"uncategorized","slug":"/blog/uncategorized/office-hours"},{"title":"Node v0.4.3","author":"ryandahl","date":"2011-03-19T05:17:59.000Z","category":"release","slug":"/blog/release/v0.4.3"},{"title":"npm 1.0: The New 'ls'","author":"Isaac Schlueter","date":"2011-03-18T06:22:17.000Z","category":"npm","slug":"/blog/npm/npm-1-0-the-new-ls"},{"title":"Welcome to the Node blog","author":"ryandahl","date":"2011-03-18T03:17:12.000Z","category":"video","slug":"/blog/video/welcome-to-the-node-blog"}]}
```

Comment:
This file shouldn't be in the diff. You might be missing the pre-commit hooks on your branch?